templates/connect-inject-deployment.yaml

```diff
@@ -84,6 +84,7 @@ spec:
                 -envoy-image="{{ .Values.global.imageEnvoy }}" \
                 -consul-k8s-image="{{ default .Values.global.imageK8S .Values.connectInject.image }}" \
                 -listen=:8080 \
+                -enable-health-checks="{{ .Values.connectInject.healthChecks.enabled }}" \
```

Comment:
Should this be something like `-enable-health-check-sync` so it's not thought to be enabling health checks on the injector itself? Or is that unnecessary?
cc @hashicorp/consul-eco-platform

Reply:
That makes sense to me. Another option is to call it `-enable-health-checks-controller`.

Reply:
I like `-enable-health-checks-controller`, that is a great one